code/modules/clothing/suits/wintercoats.dm

```diff
@@ -215,8 +216,15 @@
 	desc = "A lab-grade winter coat made with acid resistant polymers. For the enterprising chemist who was exiled to a frozen wasteland on the go."
 	icon_state = "coatchemistry"
 	inhand_icon_state = "coatchemistry"
+/obj/item/clothing/suit/hooded/wintercoat/medical/chemistry/Initialize(mapload)
+	. = ..()
+	allowed += list(
+		/obj/item/storage/bag/chemistry,
+	)
+
 	hoodtype = /obj/item/clothing/head/hooded/winterhood/medical/chemistry
 
+
```

Comment:
why

Reply:
Fixed, but will post it later as I have no Ethernet currently and writing from phone